src/helm/proxy/clients/vertexai_client.py

```diff
@@ -31,6 +32,12 @@ def __init__(
         self.tokenizer = tokenizer
         self.tokenizer_name = tokenizer_name
 
+        # VertexAI's default safety filter is overly sensitive, so we disable it.
+        self.safety_settings: Dict[HarmCategory, SafetySetting.HarmBlockThreshold] = {
```

Comment:
Perhaps this should be a flag?  I imagine when we evaluate the safety of a model, we might be interested in what happens when there's a safety model in place?

Reply:
I'll add a flag in a follow up PR.